cfgov/unprocessed/css/molecules/inset.less

```diff
@@ -46,6 +68,83 @@
         margin-right: unit( @grid_gutter-width / @base-font-size-px, em );
     });
 
+    &__image {
+        float: right;
+
+        div.m-inset_image {
```

Comment:
Why chain to the `div`? If we're keeping decorative images it should be a modifier like `m-inset_image__decorative`